src/main/host/thread.c

```diff
@@ -212,6 +212,11 @@ SysCallCondition* thread_getSysCallCondition(Thread* thread) {
     return thread->cond;
 }
 
+PluginPhysicalPtr thread_getTidAddress(Thread* thread) {
```

Comment:
The call signature does not match the one below in `thread.h` (`PluginVirtualPtr` vs `PluginPhysicalPtr`).

I *think* this one should be `PluginVirtualPtr`?

This would become important if we stop typedef'ing these both to be the same as `PluginPtr`.

Reply:
Oops, good catch. Yeah, I was thinking of making these separate types to help avoid these mistakes at the same time as improving the virtual -> physical address translation. Maybe I'll see if I can knock that out today